modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -309,3 +308,16 @@ chown postgres:postgres /var/lib/containers/storage/volumes/var-pgsql
 chown wwwrun:www /var/lib/containers/storage/volumes/var-spacewalk
 ----
 
+
+
+== Logical Volume Management (LVM)
+
+
+
+For all kind of virtual machines (VM) you do not need LVM because the disk is virtual and thus can be increased.
+With containers {productname} now has [path]``/var/lib/containers/storage/volumes`` as an extra disk.
+Additional disks for subvolumes are possible, and [command]``mgr-storage-server`` does it for the database volume [literal]``var-pgsql``.
```

Comment:
I am sorry, but I don't understand :) 

`With containers {productname} now has [path]``/var/lib/containers/storage/volumes`` as an extra disk.`
What does this mean?

When you install SUMA, irrespective of installation method, you will not have that volumes path as an extra disk. It is just a directory.

After installation you can use `mgr-storage-server` with one or two parameters.
One parameter variant `mgr-storage-server block_device` will mode complete `volumes` to the extra disk.
Two parameter variant `mgr-storage-server block_device_1 block_device_2` will mode above mentioned `volumes` to first disk and `var-pgsql` sub-directory of it to second disk.

I think we do not need this segment here at all. Just mention that LVM generally is not needed and recommended. If users wants to use it, they need to prepare `/var/lib/containers/storage/volumes` mount point manually, possibly `/var/lib/containers/storage/volumes/var-pgsql` as well.